likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/model/LMFeedPollViewData.kt

```diff
@@ -0,0 +1,105 @@
+package com.likeminds.feed.android.core.post.model
+
+import com.likeminds.likemindsfeed.post.model.PollMultiSelectState
+import com.likeminds.likemindsfeed.post.model.PollType
+
+class LMFeedPollViewData private constructor(
+    val id: String,
+    val title: String,
+    val pollAnswerText: String,
+    val toShowResults: Boolean,
+    val options: List<LMFeedPollOptionViewData>,
+    val expiryTime: Long,
+    val isAnonymous: Boolean?,
```

Comment:
```suggestion
    val isAnonymous: Boolean,
```
This should not be nullable.